src/core/memory.d

```diff
@@ -811,9 +811,11 @@ struct GC
 }
 
 /**
- * Pure variants of C's memory allocation functions `malloc`, `calloc` and
- * `realloc`. Purity is achieved via resetting the `errno` to it's value prior
- * to being called, removing the function's global state mutation.
+ * Pure variants of C's memory allocation functions `malloc`, `calloc`, and
+ * `realloc` and deallocation function `free`.
+ *
+ * Purity for allocation is achieved via resetting the `errno` to it's value
```

Comment:
Since deallocation must also save&restore errno (see comment at `pureFree`):
Purity for allocation ->  Purity for allocation and deallocation